tests/ui/use_crate_prefix_for_self_imports.rs

```diff
@@ -0,0 +1,5 @@
+#![warn(clippy::use_crate_prefix_for_self_imports)]
+
+fn main() {
+    // test code goes here
```

Comment:
This file seems automatically generated and superfluous, as it tests nothing.

Again, I'd like to see a test case where we have an import right next to the module.